modules/@ergonode/ui/src/components/Grid/Layout/Table/GridTableLayout.vue

```diff
@@ -8,140 +8,132 @@
-                @sticky="onStickyChange" />
-        </GridTableLayoutColumnsSection>
-        <GridTableLayoutPinnedSection :is-pinned="pinnedSections[pinnedState.RIGHT]">
-            <GridActionColumn
-                v-for="(column, columnIndex) in actionColumns"
-                :style="templateRows"
-                :key="column.id">
-                <GridTableCell
-                    :row="rowsOffset"
-                    :column="orderedColumns.length + columnIndex + columnsOffset"
-                    :locked="true" />
-                <GridTableCell
-                    v-if="isBasicFilter"
-                    :row="rowsOffset + basicFiltersOffset"
-                    :column="orderedColumns.length + columnIndex + columnsOffset"
-                    :locked="true" />
-                <template v-for="(row, rowIndex) in rows">
-                    <GridActionCell
-                        :key="`${rowIds[rowIndex]}|${column.id}`"
-                        :component="actionCellComponents[column.id]"
-                        :column-index="orderedColumns.length + columnIndex + columnsOffset"
-                        :column="column"
-                        :action="row._links.value[column.id]"
-                        :row-index="rowsOffset + rowIndex + basicFiltersOffset + 1"
-                        :is-selected="isSelectedAllRows
-                            || selectedRows[rowsOffset + rowIndex + basicFiltersOffset + 1]"
-                        @action="onRowAction"
-                    />
+                                :label="column.label"
+                                :deletable="column.deletable"
+                                :sorted-column="sortedColumn"
+                                @sort="onSortColumn"
+                                @remove="onRemoveColumn" />
+                            <template v-if="isBasicFilter">
+                                <GridFilterDataCell
+                                    v-if="column.filter"
+                                    :row-index="rowsOffset + basicFiltersOffset"
+                                    :value="filters[column.id]"
+                                    :component="dataFilterCellComponents[column.filter.type]"
+                                    :column-index="columnIndex + columnsOffset"
+                                    :language-code="column.language"
+                                    :column-id="column.id"
+                                    :filter="column.filter"
+                                    @edit-filter-cell="onEditFilterCell"
+                                    @filter-value="onFilterValueChange" />
+                                <GridTableCell
+                                    v-else
+                                    :locked="true"
+                                    :row="rowsOffset + basicFiltersOffset"
+                                    :column="columnIndex + columnsOffset" />
```

Comment:
to computed

Reply:
It cannot be computed - columnIndex is comming from v-for